src/lib/markbind/src/lib/markdown-it/markdown-it-icons.js

```diff
@@ -0,0 +1,13 @@
+module.exports = require('markdown-it-regexp')(
+    /:(fa[brs]|glyphicon)-([a-z-]+):/m,
+    (match, utils) => {
+        let iconFontType = match[1];
+        let iconFontName = match[2];
+
+        if (iconFontType === 'glyphicon') {
+            return `<span aria-hidden="true" class="glyphicon glyphicon-${iconFontName}"></span>`;
+        } else {

```

Comment:
Add a comment that this is for "Font Awesome" icons? Or perhaps check that the `iconFontType` is "fas"/"far"/"fab" before rendering it as a Font Awesome icon?

Reply:
To add on, maybe use a switch statement? Since checking `fas`/`far`/`fab` within `if else` is quite long.

Reply:
The regex `:(fa[brs]|glyphicon)-([a-z-]+):` should only catch `:fab-`/`:far-`/`:fas-`/`:glyphicon-` anyway, so a check would be redundant. Will add in a comment instead!